language/de/common.php

```diff
@@ -452,6 +456,7 @@
 	'NOTIFICATION_FORUM'				=> '<em>Forum:</em> %1$s',
 	'NOTIFICATION_GROUP_REQUEST'		=> '<strong>Beitrittsgesuch</strong> von %1$s für Gruppe %2$s.',
 	'NOTIFICATION_GROUP_REQUEST_APPROVED'	=> '<strong>Beitrittsgesuch akzeptiert</strong> für Gruppe %1$s.',
+	'NOTIFICATION_METHOD_INVALID'		=> 'Die Methode „%s“ verweist nicht auf eine gültige Benachrichtigungs-Methode.',
```

Comment:
Ich finde "bezeichnen" eigentlich auch als übersetzung von "refer to" korrekt. Findet man so auch in der Liste von [leo] (http://dict.leo.org/ende/index_de.html#/search=refer%20to&searchLoc=0&resultOrder=basic&multiwordShowSingle=on). 

Alternativ wäre noch "referenzieren" möglich. Das sieht man in dem Zusammenhang auch schon mal:
> Die Methode "%s" referenziert keine gültige Benachrichtigungs-Methode

Ich finde bezeichnen aber besser.